src/domains/ecommerce-app/04_apim_ecommerce_checkout.tf

```diff
@@ -132,3 +132,12 @@ resource "azurerm_api_management_api_operation_policy" "get_carts_redirect" {
   })
 }
 
+resource "azurerm_api_management_api_operation_policy" "transaction_activation_request" {
+  api_name            = "${local.project}-ecommerce-checkout-api-v1"
+  api_management_name = module.apim.name
+  resource_group_name = local.pagopa_apim_rg
+  operation_id        = "newTransaction"
+
+  xml_content = file("./api/checkout/checkout_ecommerce/v1/_rpt_ids_value.xml.tpl")
```

Comment:
this path is wrong, maybe  `api/ecommerce-checkout/v1` ?

Reply:
right path set